includes/class-ajax-data-retrieval.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * The core plugin class file
+ *
+ * Defines the functions necessary to register our AJAX functions
+ * in WordPress that will
+ *
+ * @since      0.0.2
+ *
+ * @package    WSUWP_Plugin_iDonate_Ajax_Data
+ * @author     Blair Lierman <blair.lierman@wsu.edu>
+ */
+class WSUWP_Plugin_iDonate_Ajax_Data {
+
+	/**
+	 * Initializes the actions necessary for hooking in our AJAX methods
+	 *
+	 * @since    0.0.2
+	 */
+	public function init() {
+
+		add_action( 'wp_ajax_nopriv_wsuwp_plugin_idonate_ajax_fund_search', array( $this, 'wsuwp_plugin_idonate_ajax_fund_search' ) );
+		add_action( 'wp_ajax_wsuwp_plugin_idonate_ajax_fund_search', array( $this, 'wsuwp_plugin_idonate_ajax_fund_search' ) );
+
+	}
+
+	public function wsuwp_plugin_idonate_ajax_fund_search() {
+		$search_term = strtolower( $_REQUEST['term'] );
```

Comment:
When using `wp_ajax_`, there should be a nonce to make sure the request is coming from the right place, especially when `wp_ajax_nopriv` is in use as that can allow any unauthenticated user to make a request. See [this](https://github.com/washingtonstateuniversity/project.wsu.edu/blob/657424055f96cccbf144f16bfdd6895b8d360ddb/functions.php#L91), [this](https://github.com/washingtonstateuniversity/project.wsu.edu/blob/657424055f96cccbf144f16bfdd6895b8d360ddb/functions.php#L66), and [this](https://github.com/washingtonstateuniversity/project.wsu.edu/blob/657424055f96cccbf144f16bfdd6895b8d360ddb/js/project-create.js#L24) as a loose example of the required pieces.